src/components/HomeComponents/Home.vue

```diff
@@ -0,0 +1,143 @@
+<script setup>
+import { onMounted, ref, watch } from 'vue'
+import { useHomeStore } from '@/stores/home.js'
+import { useProfileStore } from '@/stores/profile.js'
+import homeApi from '@/api/home.js'
+import profileApi from '@/api/profile.js'
+import Svg from '@/components/common/Image.vue'
+import ErrorPopup from '@/components/common/ErrorPopup.vue'
+import Suggestions from '@/components/HomeComponents/Suggestions.vue'
+
+const error = ref(false)
+
+// Goal setter for challenge
+const count = ref(1)
+watch(count, () => {
+    if (count.value < 1) {
+        count.value = 1
+    }
+})
+
+const homeStore = useHomeStore()
+const profileStore = useProfileStore()
+const error_message = ref()
+onMounted(async () => {
+    try {
+        const response = await homeApi.trending()
+        const suggestions = await homeApi.suggested()
+        const response_user = await profileApi.profile()
+        if (response.status === 200) {
+            homeStore.trending = response.data
+        } else {
+            error.value = true
+            error_message.value = 'There was an error with our servers please try again later'
+        }
+        if (suggestions.status === 200 && response_user.status === 200) {
+            homeStore.suggestions = response.data
+            profileStore.user = response_user.data
+        } else {
+            error.value = true
+            error_message.value = 'There was an error with our servers please try again later'
+        }
+    } catch (err) {
+        console.log(err)
+    } finally {
+        /* empty */
+    }
+})
+// Main content cards
+const classes = ['row-span-2', 'col-span-2', 'col-span-2', '', '']
+</script>
+
+<template>
+    <ErrorPopup v-if="error" v-model="error" message="{{error_message}}" />
```

Comment:
If you want to pass a dynamic variable to a component like this error_message, you should use a colon instead, not curly braces.
```suggestion
    <ErrorPopup v-if="error" v-model="error" :message="error_message" />
```

Reply:
Also please use a camelCase for variable names (errorMessage). And by the way here you can use only one variable for error handling (not error and error_message). Just use a string to store your message and check if you should show your error popup using !!error_message